motor_controller/config/config.yaml

```diff
@@ -0,0 +1,3 @@
+# --- Config ---
+queue_size: 10
+usbport: "/dev/ttyACM0"
```

Comment:
Do you even use this? Also add a newline at the end

Reply:
Removed line 3 but line 2 is in use.